### DISCUSSION TITLE
Wrong date format for ar-SA locale?

### DISCUSSION THREAD

**Discussion by helletii** [js-fns/date-fns#2216] [Q&A]
All sources I've found state that the date format in Saudi Arabia is dd/MM/yyyy. For example here: https://en.m.wikipedia.org/wiki/Date_format_by_country ('listing'). Is there a reason that the format in date-fns ar-SA locale is MM/dd/yyyy?

**Comment by tan75** [comment]
Hi @date-fns/i18n-ar-sa 

Can you please help us to clarify this?

**Comment by kossnocorp** [comment]
I can see from CLDR, that the format is `yyyy/MM/dd`, so both are invalid. @date-fns/i18n-ar-ar @date-fns/i18n-ar-dz WDYT?

**Reply by helletii** [reply]
Hi, where do you see that? I found 'd/M/y' here: https://github.com/unicode-org/cldr-json/blob/master/cldr-json/cldr-dates-full/main/ar-SA/ca-gregorian.json
Is this also an RTL issue?
